website/docs/r/resource_group_cost_management_export.html.markdown

```diff
@@ -0,0 +1,109 @@
+---
+subcategory: "Cost Management"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_resource_group_cost_management_export"
+description: |-
+  Manages an Azure Cost Management Export for a Resource Group.
+---
+
+# azurerm_resource_group_cost_management_export
+
+Manages an Azure Cost Management Export for a Resource Group.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_storage_account" "example" {
+  name                = "example-storage-account"
+  resource_group_name = azurerm_resource_group.example.name
+
+  location                 = azurerm_resource_group.example.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "azurerm_resource_group_cost_management_export" "example" {
+  name                    = "example"
+  resource_group_id       = azurerm_resource_group.example.id
+  recurrence_type         = "Monthly"
+  recurrence_period_start = "2020-08-18T00:00:00Z"
+  recurrence_period_end   = "2020-09-18T00:00:00Z"
+
+  export_data_storage_location {
+    storage_account_id = azurerm_storage_account.example.id
+    container_name     = "examplecontainer"
+    root_folder_path   = "/root/updated"
+  }
+
+  export_data_definition {
+    type       = "Usage"
+    time_frame = "WeekToDate"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Cost Management Export. Changing this forces a new resource to be created.
+
+* `resource_group_id` - (Required) The id of the resource group on which to create an export.
+
+* `recurrence_type` - (Required) How often the requested information will be exported. Valid values include `Annually`, `Daily`, `Monthly`, `Weekly`.
+
+* `recurrence_period_start` - (Required) The date the export will start capturing information.
+
+* `recurrence_period_end` - (Required) The date the export will stop capturing information.
+
+* `delivery_info` - (Required) A `delivery_info` block as defined below.
+
+* `query` - (Required) A `query` block as defined below.
+
+* `active` - (Optional) Is the cost management export active? Default is `true`.
+
+---
+
+A `export_data_storage_location` block supports the following:
```

Comment:
as storage is included in the property name below
```suggestion
A `export_data_location` block supports the following:
```

Reply:
Suggestion from Tom is to parse the container resource ID instead of the storage account ID, in which case `storage_account_id` would be removed. Then I think `export_data_storage_location` is more appropriate, WDYT?

Reply:
👍 sounds good!